common/source/docs/common-cameras-and-gimbals.rst

```diff
@@ -41,7 +41,7 @@ Camera Control and GeoTagging
 
 ArduPilot allows you to :ref:`configure the camera shutter output port <common-camera-shutter-with-servo>` (servo, relay).
 
-.. note:: be sure to set the ``CAM_MIN_INTERVAL`` to be greater than the fastest the camera can take photos when using the camera trigger functions.
+.. note:: be sure to set the ``CAMx_MIN_INTERVAL`` to be greater than the fastest the camera can take photos when using the camera trigger functions.
```

Comment:
this one has been renamed to CAM1_INTRVAL_MIN (note the missing E)

Reply:
fixed..thanks